modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -3,3 +3,193 @@
+
+| `@janus-idp/backstage-plugin-tekton`
+| `@backstage-community/plugin-tekton`
+
+| `@janus-idp/backstage-plugin-tech-radar`
+| `@backstage-community/plugin-tech-radar`
+
+| `@janus-idp/backstage-plugin-tech-radar-backend`
+| `@backstage-community/plugin-tech-radar-backend`
+
+| `@janus-idp/backstage-plugin-sonarqube`
+| `@backstage-community/plugin-sonarqube`
+
+| `@janus-idp/backstage-plugin-sonarqube-backend`
+| `@backstage-community/plugin-sonarqube-backend`
+
+| `@janus-idp/backstage-plugin-lighthouse`
+| `@backstage-community/plugin-lighthouse`
+
+| `@janus-idp/backstage-plugin-nexus-repository-manager`
+| `@backstage-community/plugin-nexus-repository-manager`
+
+| `@janus-idp/backstage-plugin-topology`
+| `@backstage-community/plugin-topology`
+
+| `@janus-idp/backstage-plugin-jfrog-artifactory`
+| `@backstage-community/plugin-jfrog-artifactory`
+
+| `@janus-idp/backstage-plugin-ocm`
+| `@backstage-community/plugin-ocm`
+
+| `@janus-idp/backstage-plugin-ocm-backend`
+| `@backstage-community/plugin-ocm-backend`
+
+| `@janus-idp/backstage-plugin-rbac`
+| `@backstage-community/plugin-rbac`
+
+| `@janus-idp/backstage-plugin-redhat-argocd`
+| `backstage-community/plugin-redhat-argocd`
+
+| `@janus-idp/backstage-plugin-rbac-backend`
+| `@backstage-community/plugin-rbac-backend`
+
+| `@janus-idp/backstage-catalog-backend-module-pingidentity`
+| `@backstage-community/plugin-catalog-backend-module-pingidentity`
+
+| `@janus-idp/backstage-scaffolder-backend-module-regex`
+| `@backstage-community/plugin-scaffolder-backend-module-regex`
+
+| `@janus-idp/backstage-plugin-quay`
+| `@backstage-community/plugin-quay`
+
+| `@janus-idp/backstage-scaffolder-backend-module-sonarqube`
+| `@backstage-community/plugin-scaffolder-backend-module-sonarqube`
+
+| `@janus-idp/backstage-scaffolder-backend-module-servicenow`
+| `@backstage-community/plugin-scaffolder-backend-module-servicenow`
+
+| `@janus-idp/backstage-plugin-analytics-provider-segment`
+| `@backstage-community/plugin-analytics-provider-segment`
+
+| `@janus-idp/backstage-plugin-scaffolder-backend-module-quay`
+| `@backstage-community/plugin-scaffolder-backend-module-quay`
+|===
+
+Two plugins previously under the @janus-idp scope have moved to @red-hat-developer-hub:
+
+[cols=2,%header]
+|===
+| Plugin Name
+| {product-very-short}
+
+| `@janus-idp/backstage-plugin-bulk-import`
+| `@red-hat-developer-hub/backstage-plugin-bulk-import`
+
+| `@janus-idp/backstage-plugin-bulk-import-backend`
+| `@red-hat-developer-hub/backstage-plugin-bulk-import-backend`
+|===
+
+
+[id="feature-rhidp-5156"]
+== The catalog backend module logs plugin is enabled
+
+With this update, the `backstage-plugin-catalog-backend-module-logs` is enabled and converted to a static plugin improving performance and stability. The dynamic plugin was disabled in version `1.3`.
+
+[id="feature-rhidp-5218"]
+== {rhsso-brand-name} `7.6` is deprecated as an authentication provider
```

Comment:
@jmagak Should this not be included in the _Deprecated functionalities_ section?